.github/workflows/ContinuousIntegration.yml

```diff
@@ -0,0 +1,30 @@
+# Adapted from
+# https://github.com/marketplace/actions/run-julia-package-tests
+
+name: ContinuousIntegration
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  ContinuousIntegration:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: ['1', '1.3', '1.6', 'nightly']
+        julia-arch: [x86]
```

Comment:
Change to `x64` for now to avoid platform specific RNG issues leading to failed test cases.

Reply:
Good catch, changed now.